src/engine/textlint-engine-core.js

```diff
@@ -118,6 +125,17 @@ new TextLintEngine({
         this._setupRules();
     }
 
+    /**
+     * load filter rule manually
+     * Note: it high cost, please use config
+     * @param {string} ruleName
+     * @deprecated use Constructor(config) insteadof it
```

Comment:
it is needed for testing. but should not used in production.
Is `deprecated` correct?